File: Practice(Beginner)/C++/Chef and digits of a number.cpp
// Source: https://www.codechef.com/problems/LONGSEQ

#include <bits/stdc++.h>
using namespace std;

int main() {
    int t;
    cin >> t;
    while (t--){
        string s;
        cin >> s;
        long int zero = count(s.begin(),s.end(),'0');
        long int one = s.length()-zero;

        one == 1 or zero == 1 ? cout << "Yes" << endl : cout << "No" << endl;

    }
}

File: Practice(Beginner)/Python3/Chef_and_digits_of_a_number.py
# Source: https://www.codechef.com/problems/LONGSEQ

t = int(input())
for test in range(t):
    s = input()
    zero = s.count('0')
    one = len(s) - zero
    print("Yes") if (one == 1 or zero == 1) else print("No")
